getting-started-5.html

```diff
@@ -57,32 +57,70 @@ <h2><a class="anchor" id="installing"></a>Installing</h2>
 
 </li>
 
-<li><p class="first">Expand OMERO.insight client archive and move it to your home directory.</p>
+<div class="note">
+	<p class="note-text"><z class="bold">Windows</z>
+	<p class="note-text">From version 5.5.0, OMERO.insight has two installers, <z class="bold">.msi</z> and <z class="bold">.exe</z>.</p>
+	<p class="note-text">The <z class="bold">.msi</z> installer will deploy the application in the <z class="bold">Program Files</z> folder of Windows and OMERO.insight is available to all the users of the target machine.</p>
+	<p class="note-text">The <z class="bold">.exe</z> file installs by default in the <z class="bold">userspace apps folder</z>. This can be changed during install, see screenshots in Step 2. below. OMERO.insight installed in the userspace is only available to the user who did the installation.</p>
```

Comment:
```suggestion
	<p class="note-text">The <z class="bold">.exe</z> file installs by default in the <z class="bold">userspace apps folder</z>. This can be changed during install, see screenshots in Step 2 below. OMERO.insight installed in the userspace is only available to the user who did the installation.</p>
```